Solutions/Box/Data Connectors/azuredeploy_Box_API_FunctionApp.json

```diff
@@ -128,7 +128,7 @@
                 "alwaysOn": true,
                 "reserved": true,
                 "siteConfig": {
-                    "linuxFxVersion": "python|3.6"
+                    "linuxFxVersion": "python|3.8"
```

Comment:
If there is no activity in the box environment, Azure function app fails with following error - 
![image](https://user-images.githubusercontent.com/95622706/180737163-a46e82c2-78df-48db-a00a-39e1800bb485.png)

This error should be fixed

Reply:
fixed